WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -24,59 +50,96 @@ struct CouponDetails: View {
         GeometryReader { geometry in
             ScrollView {
                 VStack(alignment: .leading, spacing: 0) {
-                    Text(Localization.performance)
-                        .bold()
-                        .padding(Constants.margin)
-                        .padding(.horizontal, insets: geometry.safeAreaInsets)
-                    HStack(spacing: 0) {
-                        VStack(alignment: .leading, spacing: Constants.verticalSpacing) {
-                            Text(Localization.discountedOrders)
-                                .secondaryBodyStyle()
-                            Text(viewModel.discountedOrdersCount)
-                                .font(.title)
+                    // Anchor the action sheet at the top to be able to show the popover on iPad in the most appropriate position
```

Comment:
I tried moving `actionSheet` and `shareSheet` to the `Button` inside `ToolbarItem` in the toolbar, it anchors on the toolbar button but disappears after rotation 😅 this sounds like a good workaround.

<img src="https://user-images.githubusercontent.com/1945542/153335380-4b7a0247-baba-421d-ae89-4989b6844437.png" width="320"/>

Reply:
I think it makes sense for the popup to be dismissed on rotation. Were you able to open the popup again after that?
I attempted to test this on my iPad but it's on iOS 14 and the popup couldn't even show up (it's a known issue on older OS versions I think).

Reply:
Ah, I installed iOS 14.5 iPad simulators and I also couldn't see the popover at all 🤦‍♀️ I also tested in iOS 15 iPad device and simulator again, and the popover actually stayed after rotations. Looks like the SwiftUI action sheet is buggy, and we can revisit when we drop iOS 14.